### PR TITLE
Loops objectives

### PR DESCRIPTION
**Pull request by jimflores5**
Still not sure whether control flow should come before or after a deep dive with strings and lists (e.g. zero-based indexing, methods, etc.).  Our decision on this will influence the wording for a couple of the objectives.

### CODE REVIEW

File: unit02/loops.rst [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+Objectives
+----------
+
+``for`` Loops
+^^^^^^^^^^^^^
+
+Students will be able to:
+
+- Describe the role of each of the 3 components of the ``range`` type:
+  ``start``, ``stop``, ``step``.
```
Comment (gildedgardenia) [inline unit02/loops.rst]: ``stop``, and ``step``

File: unit02/loops.rst [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+- Recognize that only the ``stop`` component in ``range`` is required.
+- Explain what **zero-based indexing** means, and provide examples.
+- Write a ``for`` loop with ``range(start, stop)`` to iterate upwards
+  from 0 - n, downwards from n - 0, upwards from m - n, and downwards from
+  m - n.
+- Write a ``for`` loop with ``range(start, stop, step)`` to iterate upwards
+  or downwards in increments larger than 1 unit.
+- Iterate over strings without using index values.
+- Iterate over strings using index values.
```
Comment (gildedgardenia) [inline unit02/loops.rst:29]: We removed this from the chapter.

**Review by gildedgardenia — APPROVED** [review]
These look great!

File: unit02/loops.rst (hunk 1 above)
Comment (jimflores5) [reply unit02/loops.rst]: Done!

File: unit02/loops.rst (hunk 2 above)
Comment (jimflores5) [reply unit02/loops.rst:29]: You're right, but for the short term we kept this section in the book.  We'll need to update both after the schools move past chapter 6 this year.

Comment (jimflores5) [reply unit02/loops.rst:29]: I added an issue to the GitHub repo for the LCHS textbook.

**Pull request merged**